src/pgen/torus.cpp

```diff
@@ -427,7 +427,10 @@ void ProblemModifier(ParameterInput *pin) {
   }
 }
 
-void PostInitializationModifier(ParameterInput *pin, Mesh *pmesh) {
+void PostInitializationModifier(ParameterInput *pin, Mesh *pmesh, const bool is_restart) {
+  if (is_restart) {
+    return;
+  }
```

Comment:
Should `PostInitializationModifier` even be called for restarts?

Reply:
Yeah you're right I don't see any reason to ever call `PostInitializationModifier` for restarts -- I removed the `is_restart` parameter and now just don't call this function at all on restarts.